packages/view-types/feature-list/src/selectable-table/styles.js

```diff
@@ -29,7 +29,15 @@ export const useStyles = makeStyles(theme => ({
     },
   },
   tableCell: {
-    padding: '0 4px',
+    textAlign: 'right',
+    flexBasis: '0',
+    flexGrow: 1,
+  },
+  tableCellFirst: {
+    flex: 4.3,
+  },
+  tableCellSecond: {
+      flex: 5.7,
```

Comment:
```suggestion
    textAlign: 'left',
    flexBasis: 0,
    flexGrow: 1,
```


Prefer not to hard-code assumptions about the sizes of the text in the columns.